### PR TITLE
Fix gpfdist  makefile to override CPPFLAGS and incorrect output for ssl enabled

### PR DESCRIPTION
**Pull request by Chibin**
This fixes the issue:
* Error: -m max row length must be between 32KB and 1MB
* "Serving HTTP on port ..." message was not being printed

### CODE REVIEW

**Comment by jimmyyih** [comment]
:+1:

**Comment by Chibin** [comment]
@asubramanya @lij55 Can you please take a look? Added the needed line for ssl.

**Comment by asubramanya** [comment]
LGTM :+1:

File: src/bin/gpfdist/Makefile [hunk 1]
```diff
@@ -26,11 +26,11 @@ else
   APR_LIB=$(shell $(with_apr_config) --link-ld --libs)
 endif
 
-ifneq ($(PORTNAME),win32)
-  CPPFLAGS += -DGPFXDIST
-else
+ifeq ($(PORTNAME),win32)
```
Comment (lij55) [inline src/bin/gpfdist/Makefile]: A question that is not related with this PR: why we don't support GPFXDIST on Win32 (besides libyaml library)?

**Comment by lij55** [comment]
+1, thanks!

File: src/bin/gpfdist/Makefile (hunk 1 above)
Comment (Chibin) [inline src/bin/gpfdist/Makefile]: > A question that is not related with this PR: why we don't support GPFXDIST on Win32 (besides libyaml library)?

That I'm not too sure... someone with more knowledge about gpfdist may be able to answer this.

**Comment by Chibin** [comment]
pushed:
https://github.com/greenplum-db/gpdb/commit/d9e94bc1331958b7b5bb486202aed41f0f55cd60
